backend/otel/extract.go

```diff
@@ -127,169 +132,156 @@ func extractFields(ctx context.Context, params extractFieldsParams) (extractedFi
 		}
 	}
 
-	attrs := mergeMaps(
+	originalAttrs := mergeMaps(
 		resourceAttributes,
 		spanAttributes,
 		eventAttributes,
 		scopeAttributes,
 		logAttributes,
 	)
 
-	// process potential syslog message
-	if len(fields.logBody) > 0 && fields.logBody[0] == '<' {
-		extractSyslog(&fields, attrs)
+	if val, ok := originalAttrs[highlight.DeprecatedSourceAttribute]; ok {
```

Comment:
Moved this extraction up early such that we know if this is a frontend or backend source.